plugins/item_tasks/web_client/templates/paginateTasksWidget.pug

```diff
@@ -3,6 +3,7 @@
 
 .list-group.g-task-list-container
   each task in tasks
-    a.list-group-item.g-execute-task-link(href=`#item_task/${task.id}/run`)
+    a.list-group-item.g-execute-task-link(
+      href=hyperlinkCallback && hyperlinkCallback(task), data-task-id=task.id)
```

Comment:
I couldn't get it on a new line without a "Invalid attribute separator found" error.